server/utils/nunjucksSetup.ts

```diff
@@ -39,4 +39,41 @@ export default function nunjucksSetup(app: express.Express, path: pathModule.Pla
   )
 
   njkEnv.addFilter('initialiseName', initialiseName)
+
+  njkEnv.addFilter(
+    'setSelected',
+    (items, selected) =>
+      items &&
+      items.map((entry: { value: string }) => ({
+        ...entry,
+        selected: entry && entry.value === selected,
+      })),
+  )
+
+  njkEnv.addFilter('addDefaultSelectedValue', (items, text, show) => {
+    if (!items) return null
+    const attributes: { hidden?: string } = {}
+    if (!show) attributes.hidden = ''
+
+    return [
+      {
+        text,
+        value: '',
+        selected: true,
+        attributes,
+      },
+      ...items,
+    ]
+  })
+
+  njkEnv.addFilter('findError', (array, formFieldId) => {
+    if (!array) return null
+    const item = array.find((error: { href: string }) => error.href === `#${formFieldId}`)
+    if (item) {
+      return {
+        text: item.text,
+      }
+    }
+    return null
+  })
```

Comment:
I would like to refactor this file to make nunjucks helpers more easily unit testable. This is fine for now